ui/src/components/StatusSwitch.jsx

```diff
@@ -12,43 +14,42 @@ const StatusSwitch = () => {
       window.config.env.REACT_APP_NOPAS_ENDPOINT
     );
 
-    axios.get(stateUrl)
-    .then(rsp => setIsRunning(rsp.data))
-    .catch(function (error) {
-      console.log(error);
-    })
+    axios
+      .get(stateUrl)
+      .then(rsp => setIsRunning(rsp.data))
+      .catch(function(error) {
+        console.warn(error);
+      });
```

Comment:
`suspense` feature for data fetching:
https://medium.com/react-in-depth/why-react-suspense-will-be-a-game-changer-37b40fea71ec

Reply:
okay, for now ill undo the suspense change since there isnt a need for the data fetching.